### PR TITLE
Add wrangler d1 export command

### PR DESCRIPTION
**Pull request by vy-ton**
- Reorder `Reference` section to group related topics and stack rank relative developer importance

### CODE REVIEW

**Comment by vy-ton** [comment]
fyi @geelen @rozenmd

File: content/d1/build-with-d1/import-data.md [hunk 1]
```diff
@@ -105,6 +99,35 @@ Once you have run the above command, you will need to edit the output SQL file t
...
+```sh
+npx wrangler d1 export <database_name> --remote --output=./schema.sql --no-data=true
+```
+
+To export only D1 table schema:
+```sh
+npx wrangler d1 export <database_name> --remote --table=<table_name> --output=./schema.sql --no-data=true
+```
+
+### Known limitations
```
Comment (vy-ton) [inline content/d1/build-with-d1/import-data.md:126]: @geelen anything else to note here?

**Comment by vy-ton** [comment]
@elithrar Can you approve before Monday?

**Review by elithrar — APPROVED** [review]

**Comment by elithrar** [comment]
@vy-ton some non-D1 file changes require a PCX or workers team reviewer to approve

File: content/workers/wrangler/commands.md
Comment (elithrar) [inline content/workers/wrangler/commands.md:1]: This one needs a review from @crwaters16 or someone in PCX as it's outside of the D1 docs.

cc @irvinebroque who may be able to stamp.

**Review by elithrar — APPROVED** [review]

**Review by irvinebroque — APPROVED** [review]

File: content/workers/wrangler/commands.md
Comment (irvinebroque) [reply content/workers/wrangler/commands.md:1]: I can't currently, but moving forward will with https://github.com/cloudflare/cloudflare-docs/pull/13699

Comment (elithrar) [reply content/workers/wrangler/commands.md:1]: This is now fixed

**Pull request merged**